templates/data/postgresql/support.html

```diff
@@ -0,0 +1,731 @@
+        {{ image (
+          url="https://assets.ubuntu.com/v1/1ff056bf-postgres-logo.png",
+          alt="PostgreSQL logo",
+          width="852",
+          height="216",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow">
+        <h1>
+          Comprehensive PostgreSQL support,<br />
+          from open source experts
+        </h1>
+        <p>
+          Our SLA-backed support options range from security maintenance for
+          your debs, snaps and containers up to managed PostgreSQL on your
+          private infrastructure or your cloud tenants.
+        </p>
+        <p>
+          Our support goes beyond the PostgreSQL server to cover the extensions
+          and tools you typically install with PostgreSQL, such as timescale and
+          pgbackrest.
+        </p>
+      </div>
+      <div class="p-cta-block u-no-padding--bottom">
+        <a
+          class="p-button--positive"
+          aria-controls="data-relational-dbs-modal"
+          href="/data/postgresql#get-in-touch"
+        >
+          Contact us
+        </a>
+        <a href="https://assets.ubuntu.com/v1/ee2ab4fd-PostgreSQL%20datasheet.pdf">
+          Download the datasheet&nbsp;&rsaquo;
+        </a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--25-75">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2 class="p-text--small-caps">
+        Why choose Canonical<br />for PostgreSQL support
+      </h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item u-no-padding">
+          <p class="p-heading--2">
+            Up to 10 years of support and security maintenance
+          </p>
+        </li>
+        <li class="p-list__item u-no-padding">
+          <p class="p-heading--2">
+            More than 100 PostgreSQL extensions and tools
+          </p>
+        </li>
+        <li class="p-list__item u-no-padding">
+          <p class="p-heading--2">Experts in the full stack</p>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="p-equal-height-row--wrap has-divider-3">
+    <div class="p-equal-height-row__col">
+      <div class="p-equal-height-row__item">
+        <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/6137cdec-security%20maintenance.png",
+            alt="",
+            width="284",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-image-container__image"}
+            ) | safe
+          }}
+        </div>
+      </div>
+
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+
```

Comment:
please remove the empty line and in similar places below

Reply:
Fixed